pkg/reconciler/ingress/ingress.go

```diff
@@ -526,46 +526,37 @@ func isIngressPublic(ing *v1alpha1.Ingress) bool {
 }
 
 // areVirtualServicesReady checks if a virtual service has a status, and if so if it's ready.
-// the return values are (hasStatus, ready, error), where
+// The return values are (hasStatus, ready), where:
 //	hasStatus indicates whether the virtualService has a status field
 //	ready indicates whether it's been reconciled and able to receive requests
-//	error contains any errors that occurred during this
-func (r *Reconciler) areVirtualServicesReady(ctx context.Context, vses []*v1alpha3.VirtualService) (bool, bool) {
+func (r *Reconciler) areVirtualServicesReady(ctx context.Context, vses []*v1alpha3.VirtualService) (hasStatus bool, ready bool) {
 	logger := logging.FromContext(ctx)
 
-	var errs []error
-	hasStatus := true
-	ready := true
 	for _, vs := range vses {
-		_hasStatus, _ready, err := r.isVirtualServiceReady(ctx, vs)
+		hasStatus, ready, err := r.isVirtualServiceReady(ctx, vs)
 		if err != nil {
-			errs = append(errs, err)
+			// Log errors here, but don't return them.
+			// If an error occurred while checking VirtualService status, we'll just default to probing.
+			logger.Warnf("error occurred while checking virtual service status: %w", err)
```

Comment:
```suggestion
			logger.Warnf("Error occurred while checking virtual service status: %v", err)
```
Logger does not work with `%w`, in fact only `fmt.Errorf` understands it :)

Reply:
haha good point, I didn't introduce this line obviously but might as well fix